server.py

```diff
@@ -366,21 +366,21 @@ def do_get_response(tx_id):
 
 @app.route('/queue', methods=['POST'])
 def do_queue():
-    tx_id = request.get_json(force=True)['id']
+    tx_id = request.get_json(force=True)['tx_id']
```

Comment:
We should probably use a try except block here to handle exceptions where there is no tx_id present.